### DISCUSSION TITLE
Renaming the dudes

### DISCUSSION THREAD

**Discussion by mariusgreuel** [avrdudes/avrdude#847] [General]
> I will say that I'm not a fan of turning the "dude" part of avrdude into "dudes" - implying a bunch of dudes that work on AVR's, which may be true

Being the one who created the org, I feel a little bit embarrassed, now that you say it. Not sure whether ladyada will dump `avrdude` in favor of `avrgirl` because of this, but we surely do not want to potentially offend anyone. That said, the org is barely a month old, it is easily renamed, google will catch up in a second, and the consequences are not that bad:
https://docs.github.com/en/organizations/managing-organization-settings/renaming-an-organization

So, should we consider a different name? If yes, soon would be better than a year from now.

What I liked about `avrdudes` was that it potentially can host other repos than just `avrdude`. How about something more neutral, such as `avrfriends`?

**Comment by dl8dtl** [comment]
I'm not so sure it's worth the effort.
And yes, I thought about putting AVaRICE and avr-libc under the same umbrella, eventually.

**Comment by MCUdude** [comment]
If the plan is to host other AVR related tools in the future, you could use "avrtools". Seems to be avaliable!

**Reply by dl8dtl** [reply]
Do you really think it's worth the while? People seem to have got used to the new location now.

**Reply by mcuee** [reply]
Indeed, no need to change any more.
